frontend/.vscode/settings.json

```diff
@@ -9,4 +9,5 @@
     "titleBar.activeBackground": "#006FD6",
     "titleBar.activeForeground": "#ffffff"
   },
-}
\ No newline at end of file
+  "scss.lint.unknownAtRules": "ignore"
```

Comment:
Just FYI this is because the Tailwind @ rules were causing VSCode warnings and errors